<?xml version="1.0"?>

<launch>
  <arg name="frame"/>
  <arg name="worldFrame" default="world"/>

  <node name="time_optimal_controller" pkg="crazyflie_controller" type="time_optimal_controller" output="screen">
    <param name="frame" value="$(arg frame)" />
    <param name="worldFrame" value="$(arg worldFrame)" />
    <rosparam command="load" file="$(find crazyflie_controller)/config/PID_values.yaml" />
  </node>
</launch>
